.github/workflows/build.yml

```diff
@@ -0,0 +1,172 @@
+name: Build
+
+on:
+  push:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build-web:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        name: Clone
+      - uses: actions/setup-node@v1
+        name: Setup Node
+        with:
+          node-version: "*"
+      - name: Install Yarn
+        run: corepack enable
+      - name: Install npm dependencies
+        run: yarn install
```

Comment:
Is `--immutable` implied in GitHub actions?

Reply:
I will add this